lib/helper/QubitHelper.php

```diff
@@ -37,6 +37,10 @@ function render_field($field, $resource = null, array $options = [])
     $div = null;
     $culture = sfContext::getInstance()->user->getCulture();
 
+    if (isset($options['pattern'])) {
+        $options['pattern'] = '^[a-zA-Z][a-zA-Z0-9\-_]*$';
```

Comment:
@melaniekung @anvit, this could be an issue later on if someone uses a different pattern before calling the helper. It shouldn't be needed if it's already set and it may modify its value.

Reply:
thank you radda! addressed in new PR: https://github.com/artefactual/atom/pull/1707